rfcs/text/0000-rfc-process.md

```diff
@@ -0,0 +1,172 @@
+
+This early *lightweight* and *egalitarian* approach enabled the DCP to achieve rapid velocity and incorporate a diverse set of innovative design concepts.
+
+As DCP continues to mature and scale, our community can more easily contribute and learn when there is a formal and consistent **Request for Comments (RFC)** process to propose enhancements to software or governance which are then maintained in a central repository.
+
+## Author(s)
+*DCP PM*
+
+**Note:** *This proposal is based on ideas from the **HCA DCP Technical Decision-Making** white paper written by Bruce Martin with contributions from Tony Burdett, Laura Clarke, Brian O’Connor, Brian Raymor, and Tim Tickle.*
+
+## Shepherd
+[Brian Raymor](brianraymor@chanzuckerberg.com)
+
+## Motivation
+
+There are a number of limitations with our current process which need to be addressed:
+
+- For community members who want to propose enhancements to either technical software or governance, it is often unclear where to start - *for example, which HCA slack channel or project member to approach with a proposal.*
+
+  - There is no common template for design proposal submissions.
+
+  - There is no common process describing how the community reviews and approves submitted proposals.
+
+  - There is no formal assistance to guide a contributor through the process. 
+
+- New community members experience difficulties when _onboarding_. There is no curated repository of up-to-date design documents to assist with learning about the software. 
+
+- To ensure consistency of the architectural design, senior technical leaders must monitor multiple work streams or hope that they are notified of potential technical changes - *this is a significant concern for cross-cutting issues such as security or the data model*.
+
+To address these needs, the DCP RFC process defines a transparent and standard method for community members to propose and build consensus around substantial enhancements to DCP software or governance.
+
+### When is an RFC required?
+
+Any proposal that would benefit from additional review or design before being implemented is a good candidate for an RFC.
+
+- An RFC is required if the implemented design would be described in written communication to the community.
+- An RFC is required if a design impacts multiple DCP projects.
+- An RFC is required if a technical initiative (refactoring, major architectural change) impacts the community.
+- A RFC is required for all changes to the DCP formal governance, including but not limited to oversight, decision-making, conflict resolution, and community processes such as charters and RFCs. For more background on governance, see the **Model C: Delegated Governance** section in [Organization & Structure of Open Source Software Development Initiatives](https://dash.harvard.edu/bitstream/handle/1/30805146/2017-03-24_governance.pdf).  
+
```

Comment:
See my general comments here. I think this section could use some tightening. I find "Any proposal that would benefit from additional review or design" rather broad and I think I would prefer an additional typing mechanism (possibly with different sections based on the type). In our current development phase,

Reply:
I just posted this after reviewing on the plane, and I see @briandoconnor has raised a similar idea of informational RFCs whilst I was in transit :)

I talked about:
> ...three different scenarios (roughly, enhancements, framework/contract definitions, implementation roadmaps)
and the second and third types here could definitely be covered by informational RFCs